src/js/controller/model.js

```diff
@@ -229,6 +232,9 @@ define([
             // Mute the video if autostarting on mobile. Otherwise, honor the model's mute value
             _provider.mute(this.autoStartOnMobile() || _this.get('mute'));
 
+            // Set the playback rate to be the value that the provider supports and will play at
+            this.set('playbackRate', _provider.setPlaybackRate(_this.get('defaultPlaybackRate')));
```

Comment:
I don't like the pattern of returning a value from a setter. It's an extra line of code, but with SRP in mind:
```
_provider.setPlaybackRate(_this.get('defaultPlaybackRate'));
this.set('playbackRate', _provider.getPlaybackRate());
```